modules/Rock.cmake

```diff
@@ -49,6 +49,29 @@ function(rock_add_compiler_flag_if_it_exists FLAG)
     endif()
 endfunction()
 
+macro(rock_add_target_properties _target _name)
+    get_target_property(_properties ${_target} ${_name})
+    if(NOT _properties)
+        set(_properties ${ARGN})
+    else()
+        list(APPEND _properties ${ARGN})
```

Comment:
I think that you can safely append to a non-existing variable (i.e. no need for the if/else)